webpack.config.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
+
+/** @type {import("webpack").Configuration} */
+module.exports = {
+    mode: "none",
+    entry: {
+        no_unsanitize: ["./web.js"],
+    },
+    output: {
+        filename: "[name].js",
+        library: "[name]",
+        libraryTarget: "umd",
+        globalObject: "this",
+    },
+    module: {
+        rules: [
+            {
+                test: /\.m?js$/u,
+                loader: "babel-loader",
+                options: {
+                    presets: [
+                        ["@babel/preset-env", {
+                            debug: true, // ← to print actual browser versions
+
+                            /*
+               * We want to remove `transform-unicode-regex` convert because of https://github.com/eslint/eslint/pull/12662.
+               *
+               * With `>0.5%`, `@babel/preset-env@7.7.6` prints below:
+               *
+               *     transform-unicode-regex { "chrome":"49", "ie":"11", "safari":"5.1" }
+               *
+               * So this excludes those versions:
+               *
+               * - IE 11
+               * - Chrome 49 (2016; the last version on Windows XP)
+               * - Safari 5.1 (2011-2013; the last version on Windows)
+               */
```

Comment:
I'm sure that for the people developing eslint this big inline comment in their webpack config file has a meaning and is likely pointing something they would like to happen and change this preset-env config accordingly, but for us in this repo it may not be providing that much value in its current form, instead we may want to just point to our future selves that this preset-env config is meant to be possibly kept in sync with the one in the eslint repo.

e.g. something like:

```
// NOTE: keep the preset-env targets in sync with the one
// used in https://github.com/eslint/eslint 
targets: ">0.5%, not chrome 49, not ie 11, not safari 5.1",
```

This is shorter and it does tell us what we should be possibly doing about it over the time.